assets/stylesheets/components/_icon.scss

```diff
@@ -0,0 +1,7 @@
+.icon-sprite {
+  display: none;
```

Comment:
A more bulletproof way is to use the [`hidden`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) attribute in the layout:

```
<div class="icon-sprite" hidden="true">
```

Reply:
@olivierobert added [75a42c4](https://github.com/carryall/go-google-scraper-challenge/pull/25/commits/75a42c4bc17e92a14d8219b3ea5ecead06b7b6d6)